### PR TITLE
[ADD] travis/cfg/travis_run_pylint_pr.cfg

### PR DESCRIPTION
**Pull request by martintorresvm**
This could work but need to be tested.
Note that the code is arbitrary choosed as to indentify message
not really exist by Pylint.
	modified:   travis/cfg/travis_run_pylint_pr.cfg

### CODE REVIEW

**Comment by martintorresvm** [comment]
To be Tested.

But please review @elicoidal @seb-elico

File: travis/cfg/travis_run_pylint_pr.cfg [hunk 1]
```diff
@@ -11,7 +11,8 @@ cache-size=500
 [ODOOLINT]
 readme_template_url="https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst"
 manifest_required_author="Elico Corp"
-manifest_required_keys=license,support
+manifest_required_support="support@elico-corp.com"
```
Comment (seb-elico) [inline travis/cfg/travis_run_pylint_pr.cfg:14]: @victormartinelicocorp I'm not sure this is enough. You mean the check of value of the key is triggered by some naming convention? I would assume that somewhere, there's something to trigger this check on this field. IMO we should check all occurrences of `manifest_required_author` to see if there's a trigger and replicate that mechanism. If no other occurrence, then it means you're right about the automatic trigger based on name convention.

**Review by seb-elico — CHANGES_REQUESTED** [review]

**Comment by lonelysun** [comment]
@seb-elico 
 `manifest-required-author - C8101` is necessary ？
This still make Szmig repo travis red, 

> __manifest__.py:5: [C8101(manifest-required-author), ] One of the following authors must be present in manifest: 'Odoo Community Association (OCA)'

cc @victormartinelicocorp

**Comment by martintorresvm** [comment]
I would close this PR and rebase our repository fork which is out to date with OCA one
`This branch is 12 commits ahead, 28 commits behind OCA:master. `
what you think @seb-elico @lonelysun ?

cc @elicoidal

**Pull request closed without merging**